crates/iota-graphql-rpc/src/types/transaction_block/mod.rs

```diff
@@ -70,13 +70,13 @@ pub(crate) enum TransactionBlockInner {
         stored_tx: StoredTransaction,
         native: NativeSenderSignedData,
     },
-    /// A transaction block that has been executed via executeTransactionBlock
-    /// but not yet indexed.
-    Executed {
-        tx_data: NativeSenderSignedData,
-        effects: NativeTransactionEffects,
-        events: Vec<NativeEvent>,
+    /// A transaction block that has been executed but not yet indexed,
+    /// stored in the optimistic transactions table.
+    ExecutedOptimistically {
```

Comment:
I think the `Optimistically` part it unnecessary. It is now implicit that execution means that we store and index the transaction, but without checkpoint information. This can be made clear to the doc. The internal detail about the `optimistic_transactions` table can be also abstracted on this level of the API.
Something like: 


```suggestion
    /// A transaction block that has been executed and indexed without checkpoint information.
    Executed {
```

Reply:
Done in f2ada572b7933d4a3941ca966d8273b5a0b86aa9